llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1025,6 +1024,39 @@ void VPInstruction::print(raw_ostream &O, const Twine &Indent,
 }
 #endif
 
+Value *VPInstructionWithType::generate(VPTransformState &State) {
+  State.setDebugLocFrom(getDebugLoc());
+  assert(vputils::onlyFirstLaneUsed(this) &&
+         "Codegen only implemented for first lane.");
+  switch (getOpcode()) {
+  case Instruction::SExt:
+  case Instruction::ZExt:
+  case Instruction::Trunc: {
+    // Note: SExt/ZExt not used yet.
```

Comment:
Then leave them as default until they are, to avoid dead code? Admittedly unrelated.